package.json

```diff
@@ -14,7 +14,7 @@
     "axios-retry": "^4.5.0",
     "chainlist-rpcs": "^0.5.226",
     "dotenv": "^17.2.2",
-    "sablier": "^1.1.0"
+    "sablier": "1.2.0-beta.1"
```

Comment:
We'll have to bump this to the latest version, which contains the ABIs and the correct protocol versions